lib/src/lifecycle_module.dart

```diff
@@ -237,6 +237,13 @@ abstract class LifecycleModule implements DisposableManager {
     if (_childModules.contains(newModule)) {
       return new Future(() {});
```

Comment:
You didn't do this (I may have), but a better way is `Future.value(null)`, could we go ahead and change it while you're fiddling in this file?